OJP/OJP_RequestSupport.xsd

```diff
@@ -161,11 +161,33 @@
 			<xs:element name="Place" type="PlaceStructure" maxOccurs="unbounded"/>
 		</xs:sequence>
 	</xs:complexType>
+	<xs:complexType name="OperatorsStructure">
+		<xs:annotation>
+			<xs:documentation>Structure providing a collection of operators.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="OperatorsStructure" type="siri:AffectedOperatorStructure" maxOccurs="unbounded">
```

Comment:
Why does the OperatorsStructure contain a OperatorsStructure?
What is the difference between `OperatorsStructure`, `OperatorsRefStructure` and `OperatorsRefStructure`?

Reply:
OperatorsStructure is a container for full Operators. OperatorsRefStructure is only a list of references. I simplified it a bit and removed the problem with OperatorsStructure containing OperatorsStructure. Correct OperatorsStructure contains Operator(s)